docs/src/manual/construction.md

```diff
@@ -14,31 +12,52 @@ interval(Inf) # not valid since infinity is not part of an interval
 interval(3, 2) # not valid since the lower bound is strictly greater than the upper bound
 ```
 
-The submodule `IntervalArithmetic.Symbols` exports the infix operator `..` as an alias for `interval`.
+The above intervals came with a decoration. Decorations are flags, or labels, attached to intervals to indicate the status of a given interval. Decorated intervals provide valuable information on the result of evaluating a function on an initial interval.
```

Comment:
We should mention here the existence of `bareinterval` as non-guaranteed intervals, with a link to the relevant docs.